src/test/java/com/telefonica/iot/cygnus/interceptors/GroupingRuleTest.java

```diff
@@ -66,4 +68,82 @@ public void testFiwareServicePathStartsWithSlash() {
         } // try catch
     } // testFiwareServicePathStartsWithSlash
     
+    @Test
+    public void testIfGetsAreNotNull() {
+        System.out.println(getTestTraceHead("[GroupingRule.groupingRule]")
+                + "-------- Rule's attributes are not null");
+        JSONObject jsonRule = new JSONObject();
```

Comment:
Maybe this can be moved to a `createJsonRule()` function. Just in case we need to create more rules in the future.